index.js

```diff
@@ -307,6 +271,7 @@ module.exports = (robot, _, Settings = require('./lib/settings')) => {
 
     let suborg = getModifiedSubOrgConfigName(payload)
```

Comment:
What happens if more than one suborg is pushed on a commit?

Isn't only the first one synched?